### DISCUSSION TITLE
Couple Apps

### DISCUSSION THREAD

**Discussion by nxtlvl2t8k** [skiptools/skip#29] [Ideas]
I have a couple apps built in Swift that I want to try and use with skip.  One is a dice game, One is a card game.  Let me know if you are interested in helping me.

**Comment by marcprux** [comment]
We would certainly be interested in helping guide you along. But first, please be aware of Skip's current limitations, starting with [Skip Status](https://skip.tools/docs/status/) and the [SkipUI Status](https://skip.tools/docs/modules/skip-ui/). 

Since you mention games, note that we don't have much in terms of game-supporting technologies yet (no canvas drawing, no audio APIs, etc.), so it might be a little while before Skip is ready for games projects in general.
